### DISCUSSION TITLE
negative reconstructed pupil diameter response toward CSs

### DISCUSSION THREAD

**Discussion by yingjieQ** [bachlab/PsPM#722] [Q&A] [answered]
Hi all,

I'm analyzing pupil data from human fear conditioning paradigm to see if conditioned responses have developed towards CS+US- in comparison to CS-. I used all 80 trials from conditioning phase: 20 CS+US+, 20 CS+US- and 40 CS-. For preprocessing, I took all default parameters, in many sessions, the results look rather good and plausible; in a few other cases, negative reconstructed response were observed towards CS+US- and/or CS-, rarely also CS+US-, which seems not plausible. 

I wonder if the following could be the possible reasons: 1) included CS+US+ trials in glm modelling. 2) in our setup, pupil data were collected in a remote mode without chin-rest, in some earlier sessions, maybe because of distance/pupil threshold setting, recorded pupil sizes are relatively low, many samples are below 1.5 mm (the default min. physical pupil size in preprocessing) and got excluded, which made glm modelling less reliable. 3) in some earlier sessions, the US is not aversive enough (loudness of white noise) to make conditioning happen.

case1: all responses are negative
![RCon123](https://github.com/bachlab/PsPM/assets/95622862/81be76ae-f8ac-480c-8242-4b9f020648a3)

case2: US not aversive enough, conditioning not successful?
![RCon123](https://github.com/bachlab/PsPM/assets/95622862/e09d4cca-4d03-4179-b4ef-3fdde03ade26)

case3: CS- response is negative
![RCon123](https://github.com/bachlab/PsPM/assets/95622862/238864a8-5705-4935-b98e-3384fc70990b)

case4: from the same participant as in case 3, a negative dip of CS-
![LCon123](https://github.com/bachlab/PsPM/assets/95622862/a4f15809-269e-4212-81b5-cb52bb757ec7)

I appreciate your insights and suggestions on this!

Best,
Yingjie

**Comment by dominikbach** [comment, marked as answer]
1. For pupil size analysis, negative reconstructed responses are not unplausible as such. The model describes only the expected CS+/CS- difference, not the actual shape of the response. As you see in figure 1 of this paper https://doi.org/10.1111/psyp.12801, although these are all dilation responses, but they look very different between experiments. It's just their (group average) difference between CS+/CS- that is consistent. The only thing you can interpret from this type of model is the *difference* between reconstructed CS+ and CS- responses.

2. There are indeed negative differences in the plots above, but in individual participants this might be due to noisy data.

3. Based on this initial paper, you might also leave out the derivative.
